src/tracks/playabletrack/wavetrack/ui/PitchAndSpeedDialog.cpp

```diff
@@ -310,7 +310,7 @@ bool PitchAndSpeedDialog::SetClipSpeedFromDialog()
    const auto expectedEndTime =
       start + (end - start) * mOldClipSpeed / mClipSpeed;
 
-   if (expectedEndTime >= maxEndTime)
+   if (expectedEndTime > maxEndTime)
```

Comment:
`maxEndTime` is the start time of the next clip, and `expectedEndTime`, that of this clip after stretch. It's fine for them to be equal: clip boundaries abide by a closed-begin/open-end rule.